uint8 idle = 0
uint8 configuring = 1
uint8 executing = 2
uint8 home_automation_state